Library/Formula/postgresql.rb

```diff
@@ -1,9 +1,11 @@
 class Postgresql < Formula
   homepage "http://www.postgresql.org/"
+  revision 1
 
   stable do
     url "http://ftp.postgresql.org/pub/source/v9.4.0/postgresql-9.4.0.tar.bz2"
     sha256 "7a35c3cb77532f7b15702e474d7ef02f0f419527ee80a4ca6036fffb551625a5"
+    version "9.4"
```

Comment:
I tried:

``` 
def major_version
    version.to_s[/^\d\.\d/]
end
system "#{bin}/initdb", "#{var}/postgres-#{major_version}"
```

and 

```
def version_major
    version.to_s[/^\d\.\d/]
end
system "#{bin}/initdb", "#{var}/postgres-#{version_major}"
```

and what we do for ` gcc `

```
  def version_suffix
    version.to_s[/^\d\.\d/]
  end
system "#{bin}/initdb", "#{var}/postgres-#{version_suffix}"
```

And each time I run headlong into:

```
~> brew info /Users/Dominyk/Documents/Dominyk/Dev/Github/homebrew/Library/Formula/postgresql.rb
Error: undefined local variable or method `version_suffix' for Formulary::Formulae::Postgresql:Class
```

Welp :confused: